EliteDangerous/EliteDangerous/RoutePlotter.cs

```diff
@@ -29,18 +29,40 @@ public class RoutePlotter
         public Point3D Coordsto;
         public string FromSystem;
         public string ToSystem;
-        public int RouteMethod;
+        public Metric RouteMethod;
         public bool UseFsdBoost;
 
         // METRICs defined by systemclass GetSystemNearestTo function
-        public static string[] metric_options = {
-            "Nearest to Waypoint",
-            "Minimum Deviation from Path",
-            "Nearest to Waypoint with dev<=100ly",
-            "Nearest to Waypoint with dev<=250ly",
-            "Nearest to Waypoint with dev<=500ly",
-            "Nearest to Waypoint + Deviation / 2"
-        };
+        public static string GetMetricName(Metric metric)
+        {
+            switch (metric)
+            {
+                case Metric.IterativeNearestWaypoint:
+                    return "Nearest to Waypoint";
+                case Metric.IterativeMinDevFromPath:
+                    return "Minimum Deviation from Path";
+                case Metric.IterativeMaximumDev100Ly:
+                    return "Nearest to Waypoint with dev<=100ly";
+                case Metric.IterativeMaximumDev250Ly:
+                    return "Nearest to Waypoint with dev<=250ly";
+                case Metric.IterativeMaximumDev500Ly:
+                    return "Nearest to Waypoint with dev<=500ly";
+                case Metric.IterativeWaypointDevHalf:
+                    return "Nearest to Waypoint + Deviation / 2";
+                default:
+                    throw new ArgumentOutOfRangeException(nameof(metric));
+            }
+        }
+
+        public enum Metric:int
```

Comment:
Metric:int you don't need the int. Its that by default.

Reply:
Removed the `int`